.github/CODEOWNERS

```diff
@@ -1,10 +1,8 @@
 # Default owners
 * @ovh/team-manager-dev
-yarn.lock @ovh/team-manager-core
 
 # Documentation
 *.md
-/docs/ @ovh/team-manager-core
```

Comment:
**nitpick(non-blocking):** the whole Documentation block could be removed (see L2).